### ISSUE TITLE
[SF][SuggestionItem]: A11y - Sometimes the suggestion item text is not announced

### ISSUE THREAD

**Issue by 2bno1** [SAP/ui5-webcomponents#9835]
### Bug Description

Navigating suggestion item via Arrow Up / Down key and observe the announcement of JAWS. Sometimes the text of the suggestion item is not announced by JAWS. Instead, it just announce general content: "List item {index} of {item count}".

### Affected Component

SuggestionItem

### Expected Behaviour

The text of the suggestion item should be announced by JAWS when navigating to the item.

### Isolated Example

https://stackblitz.com/edit/github-tlank3?file=package.json,src%2FApp.tsx

### Steps to Reproduce

1. Make sure JAWS is running
2. Goto https://stackblitz.com/edit/github-tlank3?file=package.json,src%2FApp.tsx
3. Type a character in input to make the suggestion items pop up
4. Press Arrow Up / Down key to navigate back / forth the suggestion items (Option 1 -> 2 -> 3 -> 2 -> 1 -> 2 -> 3)
5. Observe the announcement of JAWS

### Log Output, Stack Trace or Screenshots

![Screenshot 2024-09-10 at 10 43 48](https://github.com/user-attachments/assets/dfa3bcd1-3bf0-4539-af23-ce23e3251f6e)

### Priority

None

### UI5 Web Components Version

1.24.9

### Browser

Edge

### Operating System

Windows 11

### Additional Context

JAWS version: 2024.2403.3

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by didip1000** [comment]
Hello @SAP/ui5-webcomponents-topic-rl,

Can you please take a look at this issue? It's reproducible, and there doesn't seem to be any pattern as to when the item is read and when it's skipped.

Best,
Diana P.

**Comment by nikoletavnv** [comment]
This bug can be reproduced in the following cases: 
- When we navigate among suggestion items with equivalent length - the item text is not read
- When we switch between suggestion items with different length - the item text is read
It is also seen in web-components version 2.x and in Chrome browser. 

Note: The JAWS VPC mode should be turned off to reproduce this issue.

**Comment by nikoletavnv** [comment]
Hello colleagues, 

I could reproduce this issue in a sample without the web components: https://jsbin.com/puzuhocati/edit?html,output

Steps to reproduce: 
1. Open the sample
2. Set focus to the first input
3. Navigate with Arrow Up and Arrow Down keys to change the input value to one of the list items values. 
4. JAWS does not read all values.
5. Click on the second input and do the same - JAWS reads all values because they have different length

The problem is that when the input value is set by code, when the current value has the same length as the previous one - the text in the value is not read.  

The bug is related to known JAWS issues. 

I am closing this ticket. 

Best regards, 
Nikoleta Terzieva

**Comment by nikoletavnv** [comment]
The JAWS issue for not reading the input value when it is changed with a text of the same length can be seen here: https://github.com/FreedomScientific/standards-support/issues/379
It is reported for role="combobox" but is also reproducible for input control with no role assigned. The issue is present in different JAWS versions.
